<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>531</width>
      <height>561</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>559</x>
      <y>10</y>
      <width>231</width>
      <height>571</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>Управление списком студентов</string>
    </property>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>211</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Добавить из файла(.txt)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>211</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Добавить</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="name">
     <property name="geometry">
      <rect>
       <x>62</x>
       <y>190</y>
       <width>161</width>
       <height>28</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="inputMask">
      <string/>
     </property>
     <property name="maxLength">
      <number>80</number>
     </property>
     <property name="placeholderText">
      <string notr="true"/>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Ф.И.О</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Группа</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="group">
     <property name="geometry">
      <rect>
       <x>62</x>
       <y>230</y>
       <width>161</width>
       <height>28</height>
      </rect>
     </property>
     <property name="whatsThis">
      <string/>
     </property>
     <property name="accessibleName">
      <string/>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhDigitsOnly</set>
     </property>
     <property name="inputMask">
      <string notr="true"/>
     </property>
     <property name="text">
      <string notr="true"/>
     </property>
     <property name="maxLength">
      <number>6</number>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
     <property name="placeholderText">
      <string notr="true"/>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="marks">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>270</y>
       <width>101</width>
       <height>29</height>
      </rect>
     </property>
     <property name="maximum">
      <double>10.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>270</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Средний балл</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="earnings">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>300</y>
       <width>101</width>
       <height>29</height>
      </rect>
     </property>
     <property name="maximum">
      <number>999999</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>300</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Доход семьи</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>211</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Удалить</string>
     </property>
    </widget>
    <widget class="QPushButton" name="search">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>340</y>
       <width>211</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Поиск по фамилии</string>
     </property>
    </widget>
    <widget class="QPushButton" name="sort">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>380</y>
       <width>211</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>qsort</string>
     </property>
    </widget>
    <widget class="QPushButton" name="inform">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>540</y>
       <width>211</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>info</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="filtr1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>420</y>
       <width>231</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Фильтр 1</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="filtr2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>450</y>
       <width>231</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Фильтр 2</string>
     </property>
    </widget>
    <widget class="QPushButton" name="acceptFiltr">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>480</y>
       <width>211</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Применить фильтр</string>
     </property>
    </widget>
    <widget class="QPushButton" name="change">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>211</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Изменить</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>510</y>
       <width>211</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Сохранить</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
